CHANGELOG

```diff
@@ -7,7 +7,7 @@ New features:
 
 Enhancements:
 
-*
+* Add a sex type behind the gender
```

Comment:
There is also an important new change: gender is no more mandatory.

Reply:
indeed